SingularityUI/app/actions/api/base.es6

```diff
@@ -2,7 +2,7 @@ import fetch from 'isomorphic-fetch';
 
 const JSON_HEADERS = {'Content-Type': 'application/json', 'Accept': 'application/json'};
 
-export function buildJsonApiAction(actionName, httpMethod, opts={}) {
+export function buildJsonApiAction(actionName, httpMethod, opts={}, keyField=undefined) {
```

Comment:
Asking out of curiosity -- why the decision to explicitly default keyField to `undefined`?